source/mongodump.txt

```diff
@@ -627,17 +627,27 @@ Options
    Creates a file named :file:`oplog.bson` as part of the
    ``mongodump`` output. The ``oplog.bson`` file, located in
    the top level of the output directory, contains oplog entries that
-   occur during the ``mongodump`` operation. This file provides
-   an effective point-in-time snapshot of the state of a
-   ``mongod`` instance. To restore to a specific point-in-time
-   backup, use the output created with this option in conjunction with
-   :option:`mongorestore --oplogReplay`.
+   occur during the ``mongodump`` operation.
+   
+   The ``oplog.bson`` file is a snapshot of the data at the time the
+   file was created. To restore the data as it was when the
+   ``oplog.bson``  file was created, use :option:`mongorestore
+   --oplogReplay`. You cannot use
+   ``mongorestore`` to restore data to any other time. For example, you
+   cannot specify a request to restore data from two days ago.
    
    Without ``--oplog``, if there are write operations during the dump
    operation, the dump will not reflect a single moment in time. Changes
    made to the database during the update process can affect the output of
    the backup.
    
+   To create a consistent ``oplog.bson`` file, pause writes on the
```

Comment:
Rather than getting into this level of detail, I think we should instead say that --oplog is provided for individual replica sets, and if users want to back up sharded clusters with mongodump, they should consult https://www.mongodb.com/docs/manual/tutorial/backup-sharded-cluster-with-database-dumps/.